.github/workflows/helm-release.yaml

```diff
@@ -72,29 +87,72 @@ jobs:
         with:
           ref: ${{ inputs.ref }}
 
-      - name: Set up python
-        uses: actions/setup-python@v2
-        with:
-          python-version: "${{ inputs.python-version }}"
-
       - name: Bump version
         id: release-tag
-        uses: bakdata/ci-templates/actions/bump-version@v1.0.0
+        uses: bakdata/ci-templates/actions/bump-version@v1.5.2
         with:
           release-type: ${{ inputs.release-type }}
+          working-directory: ${{ inputs.charts-dir }}
 
-      - name: Commit and push bumpversion.cfg file
-        uses: bakdata/ci-templates/actions/commit-and-push@v1.0.0
+      - name: Download previous Helm index
```

Comment:
This and the following steps cover a lot of subtleties of gh-pages and Helm. I think a short overview explaining why all this is necessary would be helpful.

Reply:
Good point. I have added a paragraph in the readme explaining why this workaround is necessary and what it does.